lib/ansible/modules/cloud/amazon/ec2_eip.py

```diff
@@ -382,7 +481,10 @@ def main():
         release_on_disassociation=dict(required=False, type='bool', default=False),
         allow_reassociation=dict(type='bool', default=False),
         wait_timeout=dict(default=300, type='int'),
-        private_ip_address=dict(required=False, default=None, type='str')
+        private_ip_address=dict(required=False, default=None, type='str'),
+        tag_name=dict(required=False, default=None, type='str'),
```

Comment:
all three of these are the defaults so can be removed

Reply:
Thanks for the feedback @willthames, should be good now I think